tilewe/engine.py

```diff
@@ -80,9 +80,9 @@ def _play_game(self, player_to_engine: list[int]) -> tuple[list[int], list[int],
                 board.push(move) 
 
             # put scores back in original engine order 
-            winners = [ engine_to_player[x] for x in board.winners ]
+            winners = [ player_to_engine[x] for x in board.winners ]
```

Comment:
`board.winners` is the board's perspective of players not engines, so we need to map players to engines